Buildings/Controls/OBC/CDL/Logical/Proof.mo

```diff
@@ -0,0 +1,251 @@
+within Buildings.Controls.OBC.CDL.Logical;
+block Proof "Verify two boolean inputs"
+  parameter Real validDelay(
+    final quantity="Time",
+    final unit="s")
+    "Delay to valid input. The input must remain unchanged in the time";
+  parameter Real checkDelay(
+    final quantity="Time",
+    final unit="s")
+    "Delay to check if the valid inputs are the same";
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uMea
+    "Current status"
+    annotation (Placement(transformation(extent={{-220,110},{-180,150}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uCom
+    "Commanded status"
+    annotation (Placement(transformation(extent={{-220,-30},{-180,10}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput y1
+    "True: commanded status is true and current status is false"
+    annotation (Placement(transformation(extent={{160,110},{200,150}}),
+        iconTransformation(extent={{100,40},{140,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput y2
+    "True: commanded status is false and current status is true"
+    annotation (Placement(transformation(extent={{160,-110},{200,-70}}),
+        iconTransformation(extent={{100,-80},{140,-40}})));
+
+  Buildings.Controls.OBC.CDL.Logical.TrueDelay valTru(
```

Comment:
If there is a need to have them public, then they need to have good names rather than valTru and valTru1. I made them protected for now. This also needs update to the tests.

Reply:
Leaving these two instances to be public allows in the validation to show their values. For the script itself here, there is no other special need. I agree to move it to the protected section and the tests were updated.